C/common/include/asset_tracking.h

```diff
@@ -74,4 +75,24 @@ namespace std
     };
 }
 
+class ManagementClient;
+
+class AssetTracker {
```

Comment:
A doxygen style comment on the class should be part of our standard. I would like to start building doxygen documentation of the C++ classes